app/services/work_reminder_generator.rb

```diff
@@ -0,0 +1,40 @@
+# typed: false
+# frozen_string_literal: true
+
+# This creates and queues reminder emails for in-progress drafts
+class WorkReminderGenerator
+  extend T::Sig
+
+  # Sends the day's reminders about open first drafts, using default values for the notification
+  # interval unless the caller overrides the defaults with custom values. Intended to be run daily
+  # by a cron job, but the optional manual override for notification interval is useful in case a
+  # human has to manually call the method to send dropped notifications, e.g. in the event that
+  # something goes awry with the cron job on a particular day. For example, if the cron job got
+  # wedged 2 days ago, you could call this method with the the default values plus 2.
+  sig { params(first_interval: ActiveSupport::Duration, subsequent_interval: ActiveSupport::Duration).void }
+  def self.send_first_draft_reminders(first_interval: nil, subsequent_interval: nil)
+    first_interval ||= Settings.notifications.first_draft_reminder.first_interval.days
+    subsequent_interval ||= Settings.notifications.first_draft_reminder.subsequent_interval.days
+
+    Work.with_state(:first_draft, :version_draft).find_each do |work|
```

Comment:
Sorry, It looks like `version_draft` should be part of #592

Reply:
ah ok, will back that change out, no worries